libbeat/outputs/otelconsumer/otelconsumer_test.go

```diff
@@ -109,12 +109,14 @@ func TestPublish(t *testing.T) {
 		assert.NoError(t, err)
 		assert.Len(t, batch.Signals, 1)
 		assert.Equal(t, outest.BatchACK, batch.Signals[0].Tag)
-		pcommonFields := mapstrToPcommonMap(event1.Fields)
-		want, ok := pcommonFields.Get("data_stream")
-		require.True(t, ok)
-		got, ok := attributes.Get("data_stream")
-		require.True(t, ok)
-		assert.EqualValues(t, want.AsRaw(), got.AsRaw())
+
+		var subFields = []string{"dataset", "namespace", "type"}
+		for _, subField := range subFields {
+			gotValue, ok := attributes.Get("data_stream." + subField)
+			require.True(t, ok)
```

Comment:
[Suggestion]
Add a message here, it's rather difficult to understand what went wrong when the error is only "false is not true"